draft-birkholz-scitt-receipts.md

```diff
@@ -62,22 +78,30 @@ This document defines a method for countersigning of COSE_Sign1 messages when us
 
 # Common Parameters
 
-A Transparency Service backed by a Merkle tree is defined by a collection of immutable parameters, which are used by clients to communicate with the service and to verify artifacts. Each of these parameters must be established before the service operator begins to operate the service. At minimum, the following must be defined:
+Verifiers are configured by a collection of parameters 
+to identify a Transparency Service and verify its Receipts. 
+These parameters MUST be fixed for the lifetime of the Transparency Service
+and securely communicated to all Verifiers. 
+
+At minimum, these parameters include:
 
-- Service identifier: An opaque identifier (e.g. UUID) that uniquely identifies the service and all other parameters.
+- a Service identifier: An opaque identifier (e.g. UUID) that uniquely identifies the service and all other parameters.
 
-- Tree algorithm: The tree algorithm used. This document creates a registry (see {{tree-alg-registry}}) with an initial set of tree algorithms that are described in this document.
+  > is it sufficient? why putting it in every receipt then? What are those other parameters? I was expecting a public key or a certificate. 
```

Comment:
Same as in CT, it is the unique identifier to look up all other parameters. I don't see how it can be omitted in receipts. The public key / cert like all the other parameters are communicated out-of-band.

Reply:
Ok, we should then mention the identifier can be used for securely fetching all parameters.